modules/core/src/v2/baseCoin.ts

```diff
@@ -243,9 +243,9 @@ export abstract class BaseCoin {
    * The cold wallet tool uses this function to derive an extended key that is based on the passed key and seed
    * @param key
    * @param seed
-   * @returns {{key: *, derivationPath: string}}
+   * @returns {{key: string, derivationPath: string}}
    */
-  deriveKeyWithSeed({ key, seed }) {
+  deriveKeyWithSeed({ key, seed }: { key: string, seed: string }): { key: string, derivationPath: string } {
```

Comment:
I'm trying to grok this (excuse my typescript illiteracy) ... am i correct in assuming that we are just typing the input + return values here? ... cleaning up as we go?

Reply:
Yes, exactly. This defines the types of the inputs, which were just `any` before.